src/registry.rs

```diff
@@ -1163,6 +1165,12 @@ pub enum CheckKind {
     UnusedNOQA(Option<UnusedCodes>),
 }
 
+impl From<flake8_simplify::plugins::AAndNotA> for CheckKind {
+    fn from(x: flake8_simplify::plugins::AAndNotA) -> Self {
+        CheckKind::AAndNotA(x)
+    }
+}
```

Comment:
So, the idea is that once we've migrated all checks, everything in this file could be generated via a macro (apart from the code-to-kind mapping)?

Reply:
Yes that's very much the idea :)